guides/common/modules/proc_using-freeipa-credentials-to-log-in-to-the-ProjectWebUI-with-Mozilla-Firefox.adoc

```diff
@@ -1,17 +1,17 @@
-[id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{ProjectWebUI-context}-with-a-Firefox-browser_{context}"]
-= Using {FreeIPA} credentials to log in to the {ProjectWebUI} with a Firefox browser
+[id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{ProjectWebUI-context}-with-Mozilla-Firefox_{context}"]
+= Using {FreeIPA} credentials to log in to the {ProjectWebUI} with Mozilla Firefox
 
-This section describes how to use the Firefox browser to log in to your {ProjectWebUI} with your {FreeIPA} (IdM) login and password.
+This section describes how to use the Mozilla Firefox browser to log in to your {ProjectWebUI} with your {FreeIPA} login and password.
 
 .Prerequisites
```

Comment:
It's still quite a lot of prerequisites and they are all valid. But "You have a valid {FreeIPA} login and password." and "An {FreeIPA} server is running and reachable by the host." seem quite generic and applicable to majority of the procedures. I wonder if they could be mentioned once. 
Or if you could summarize it in one sentence "You have completed XYZ steps/procedures" pointing to the previous chapters (this could include also the first prerequisite.

Reply:
Yes, some of these are really not worth mentioning ("You have valid FreeIPA credentials"). And there were a lot more opportunities to make things shorter. @domiborges if you have time, please take a look at the browser procedures now and let me know if they're something we can be happy with now.